client.go

```diff
@@ -107,14 +107,15 @@ func DialSlashGraphQLEndpoint(endpoint, key string) (*Dgraph, error) {
 
 // Login logs in the current client using the provided credentials.
 // Valid for the duration the client is alive.
-func (d *Dgraph) Login(ctx context.Context, userid string, password string) error {
+func (d *Dgraph) Login(ctx context.Context, userid string, password string, namespace uint64) error {
 	d.jwtMutex.Lock()
 	defer d.jwtMutex.Unlock()
 
 	dc := d.anyClient()
 	loginRequest := &api.LoginRequest{
-		Userid:   userid,
-		Password: password,
+		Userid:    userid,
+		Password:  password,
+		Namespace: namespace,
 	}
```

Comment:
@ahsanbarkati should we instead have a new function that takes in namespace, while the `Login` uses `GalaxyNamespace`?

Reply:
Yes. That would make sense for non-multitenancy users.